src/scripts/mapper/Initialisierung.lua

```diff
@@ -22,7 +22,7 @@ end)
 enableMapInfo("Kartenmodus")
 
 function echoM(str)
-    fg(mapperconf.color)
+    fg(getConfig("mapper.config", "royal_blue"))
```

Comment:
Muss jetzt an jeder Stelle, die vormals eine Variable benutzt hat, zukünftig die Funktion, die Variable und der Standardwert genannt werden? 🤔 HIer geht es ja noch, weil es zum Glück nur eine Stelle war...

Reply:
Natürlich nicht. Wir können und sollten ihn bei mehrfacher Nutzung in einer Methode eh als local cachen. Global cachen ist aber aktuell nicht möglich, denn du kannst nie wissen ob der Wert mittlerweile noch aktuell ist.

Aber solltest schon jetzt auf den defaultValue verzichten können WENN du dir sicher bist das du irgendwann in der Session mittels ```setConfig``` einen Wert festgelegt hast. ```defaultValue``` müsste ich optional umgesetzt haben... Muss ich aber nochmal prüfen und dann dokumentieren.

Alternativ könnte man auch mit Metatabellen spielen, dann holt sich ein Skript einmal den Wert per ```getConfig``` und sobald er später diese Variable irgendwo nutzt werden alle Lese- und Schreibzugriffe unsichtbar an die Datenbank weitergeleitet. Kann ich ja mal probehalber gucken. Hab aber noch nie mit Luas Metatabellen gearbeitet. Weiß nur das es gehen sollte.

Reply:
Das ist aber totaler Blödsinn... wäre auch nicht anders als wenn man jetzt meine beiden Funktionen nutzt.

Reply:
Das mit setConfig() leuchtet ein. 
Metatabellen finde ich nicht nötig.
Tatsächlich scheint defaultValue optional und getConfig() gibt bei Fehlen also nil zurück.